### ISSUE TITLE
sequenceDiagram: autonumber

### ISSUE THREAD

**Issue by jasonlivie** [mjbvz/vscode-markdown-mermaid#52]
Hi,

I am trying to use the autonumber feature in sequenceDiagrams with Markdown Preview Mermaid support v1.4.2 and Visual Studoi Code 1.41.0. Is this feature supported because the rendering does not work for me.

thanks

**Comment by mjbvz** [comment]
Can you please share a code example of this

**Comment by jasonlivie** [comment]
```mermaid
sequenceDiagram
    autonumber
    Alice->>John: Hello John, how are you?
    loop Healthcheck
        John->>John: Fight against hypochondria
    end
    Note right of John: Rational thoughts!
    John-->>Alice: Great!
    John->>Bob: How about you?
    Bob-->>John: Jolly good!
```

**Comment by AceLondon** [comment]
@mjbvz I created a PR to upgrade mermaid to v8.4.8 which is where "autonumber" is supported.

**Comment by jasonlivie** [comment]
I just tested the 8.4.8 support / v1.4.3, works great. Thanks for your help. 

One thing though, in vscode, the autonumber appear light gray on white, the circles are light grey and the numbers are white, it makes it very hard to see depending on where they are positioned. I don't know if its possible to get a bit more contrast?

**Comment by mjbvz** [comment]
Fixed by #57

**Comment by hak33m16** [comment]
This appears to be an issue again as of v1.23.1
